README.markdown

```diff
@@ -31,7 +31,8 @@ This should give you the flexibility you need for customizing your input/output
 
 ### Building Your Phar
 
-Simply run `php package.php`, which will create the file: `build/doctrine-migrations.phar` for you.
+The PHAR is ready for [download here](http://github.com/downloads/ericclemmons/migrations/doctrine-migrations.phar), but if you still would like to build your PHAR
```

Comment:
@stof had no idea the URL was not autoupdate. If the project is run through automated tests, i.s. travis, then the PHAR can be autoupdated somewhere on a fixed URL when builds pass